src/components/container/ProviderSearch/ProviderSearch.tsx

```diff
@@ -14,20 +14,24 @@ export interface ProviderSearchProps {
     providerCategories?: string[];
     onProviderConnected?: (provider: ExternalAccountProvider) => void;
     innerRef?: React.Ref<HTMLDivElement>
-    connectExternalAccountOptions?: ConnectExternalAccountOptions
+    connectExternalAccountOptions?: ConnectExternalAccountOptions,
+    hideAddProviderOrHealthPlanButton?: boolean
```

Comment:
Prop name should probably be "hideRequestProviderButton" - the button is to request us adding one to our list - could be confusing to use "add" as one could conflate that with the user adding the account they found

Reply:
@chrisnowak I used a separator vs "or".  
I'm concerned concatenating the strings together vs translating the 3 potential values will create an output that doesn't align with how it would be naturally spoken
Do you have any thoughts with keeping it as is with the "/" or using "or" making 3 dedicated translations for
1. Request to add Providers
2. Request to add Health Plans 
3. Request to add Providers or Health Plans 

Not sure if this has come up before